app/models/government.rb

```diff
@@ -1,8 +1,12 @@
 class Government < ActiveRecord::Base
+  DISTANT_FUTURE = Date.today + 100.years
```

Comment:
:+1: to @edds comment, i've seen some really hard to trace bugs as a result of date constants. I know it doesn't matter here, but making it not a constant would set a good example for other.

(I don't _think_ this is quite common enough to be documented in the styleguide tho')